kgforge/core/wrappings/paths.py

```diff
@@ -28,10 +28,11 @@ class FilterOperator(Enum):
 
 
 class Filter:
-    def __init__(self, path: List[str], operator: str, value: Any) -> None:
+    def __init__(self, path: List[str], operator: Union[str, FilterOperator], value: Any) -> None:
         self.path: List[str] = path
         try:
-            self.operator: str = FilterOperator(operator).value
+            self.operator: str = operator.value if isinstance(operator, FilterOperator) \
+                else FilterOperator(operator).value
```

Comment:
or can you add a test case for this change?

Reply:
I made the MR because it's already currently being used as a FilterOperator in the notebooks. 
So the typing was wrong but it was working. It's just that it was doing FilterOperator(filter_operator_instance)

Reply:
Or rather, "I opened the ticket". There was no bug but the typing was misleading

Reply:
OK, well filters in the notebook example run fine, so it's enough for me.